translate/src/modules/entitieslist/components/Entity.tsx

```diff
@@ -135,6 +135,7 @@ export function Entity({
             content={entity.original}
             format={entity.format}
             search={parameters.search}
```

Comment:
I think if you do this, you don't need to use `translations_only` and pass it further:

```suggestion
            search={!parameters.search_translations_only && parameters.search}
```

Reply:
Everything else looks and works great!